config/routes.rb

```diff
@@ -1,3 +1,17 @@
 Rails.application.routes.draw do
+  get 'welcome/index'
+  get '/signup' => 'users#new'
+  post '/users' => 'users#create'
+  resources :users
+  resources :posts
+  get '/login' => 'sessions#new'
+  post '/login' => 'sessions#create'
+  get '/logout' => 'sessions#destroy'
+  root to: 'posts#index'
+
+  post '/like' => 'likes#add'
+  get '/like' => 'likes#show'
+  post '/subscribe' => 'subscriptions#add'
```

Comment:
То же самое, что и в с случае с `likes`
```suggestion
  resources :subscriptions, only: [:create, :destroy]
```